zeth_tool/README.md

```diff
@@ -1,4 +1,4 @@
-# `zeth-tool`
+# Zeth tool
```

Comment:
It may make sense to distinguish between this tool "zeth-tool", and the client "zeth tool".

Reply:
Using back ticks doesn't render properly in the Doxygen doc. I switched to `zeth-tool command` for consistency with `pot-process command`